src/App.vue

```diff
@@ -16,6 +16,9 @@
 <script>
     import WindowSize from "./components/WindowSize";
     import HeaderUSGS from './components/HeaderUSGS';
+    import gsap from "gsap";
+    import { ScrollToPlugin } from "gsap/ScrollToPlugin"; // to trigger scroll events
+    import { ScrollTrigger } from "gsap/ScrollTrigger"; // animated scroll events
     export default {
```

Comment:
Just for clarification on my end, doing this here imports to the whole app, vs within the component is just for the given section?

Reply:
Well this was a mistake to commit.  I was experimenting, but this brings up a good question.  Can we make the GSAP lib global so we don't have to import in each place we want it?  Will see if we can.